navigation.yaml

```diff
@@ -1456,6 +1456,26 @@ security:
       path: /security/thank-you
       hidden: True
 
+security-certifications:
+  title: Certifications
+  path: /security/certifications
+
+  children:
+    - title: Overview
+      path: /security/certifications
+    - title: FIPS
+      path: /security/fips
+      persist: True
+    - title: Common Criteria
+      path: /security/cc
+      persist: True
+    - title: DISA-STIG
+      path: /security/disa-stig
+      persist: True
+    - title: CIS
+      path: /security/cis
+      persist: True
+
```

Comment:
@bethcollins92 didn't we already have all this stuff in the security-certs feature branch? I remember reviewing it with you in a call

Reply:
Oh I see the problem. @petesfrench this is not correct, take a look at [this](https://github.com/canonical-web-and-design/ubuntu.com/blob/security-certifications/navigation.yaml#L1454)

Reply:
Yes, you can see [here](https://github.com/canonical-web-and-design/ubuntu.com/pull/10011/files)